### PR TITLE
EUI-4303

### PR DESCRIPTION
**Pull request by ishOsw**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4303

### Change description ###
 handling Google tag manager and dynatrace cookies when cookies are rejected or accepted for the cookie banner

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by leepavlouhmcts — DISMISSED** [review]

File: src/app/store/selectors/app.selectors.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { createSelector } from '@ngrx/store';
+import { of } from 'rxjs';
```
Comment (adnanakgun) [inline src/app/store/selectors/app.selectors.ts:2]: Is this import used in this file?

File: src/environments/environment.prod.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 export const environment = {
     production: true,
     googleAnalyticsKey: 'UA-124734893-3',
+    googleTagManagerKey: 'GTM-TQ7PKDF',
```
Comment (adnanakgun) [inline src/environments/environment.prod.ts:4]: Have you confirmed this is the key for MO? Seems to be the same as MC, that's why I ask.

File: src/environments/environment.ts [hunk 3]
```diff
@@ -6,6 +6,7 @@
 export const environment = {
     production: false,
     googleAnalyticsKey: 'UA-124734893-4',
+    googleTagManagerKey: 'GTM-TQ7PKDF',
```
Comment (adnanakgun) [inline src/environments/environment.ts:9]: Same as abobe

File: src/app/app-initializer.ts [hunk 4]
```diff
@@ -2,21 +2,23 @@ import { select, Store } from '@ngrx/store';
...
 export function initApplication(store: Store<fromApp.State>): VoidFunction {
   return () => new Promise(resolve => {
     store.dispatch(new fromApp.StartAppInitilizer());
     store.dispatch(new fromApp.LoadFeatureToggleConfig([AppConstants.FEATURE_NAMES.feeAccount,
                                                         AppConstants.FEATURE_NAMES.editUserPermissions,
                                                         AppConstants.FEATURE_NAMES.unassignedCases]));
+
     store.pipe(
-      select((state: fromApp.State) => state.appState),
+      select(fromSelectors.getAppState),
```
Comment (adnanakgun) [inline src/app/app-initializer.ts:15]: What issue does this resolve?

Comment (ishOsw) [reply src/app/app-initializer.ts:15]: resolves `ERROR: 'Unhandled Promise rejection:', 'Cannot read property 'subscribe' of undefined'`

**Review by adnanakgun — APPROVED** [review]

**Review by leepavlouhmcts — APPROVED** [review]

**Pull request merged**